### PR TITLE
Refresh groups a bit, mostly Xfce

### PR DESCRIPTION
**Pull request by hroncok**
I was looking at compos, but it got tedious, not sure if this can be somehow automated instead?

### CODE REVIEW

**Comment by hroncok** [comment]
Compos could be the data source to use but there are several challenges:

 * the groups would need to be resolved recursive as they can contain other groups
 * the groups would need to be hand picked, as there is too many
 * the groups contain binary packages, not source packages
 * some packages in the group are not tracked in portingdb but can require stuff that is tracked in portingdb (I consider this the most challenging bit.)

To summarize: a nice introductory challenge for a newcomer, but nether you or me would invest their time into this.

**Comment by encukou** [comment]
OK, probably not worth automating.
Thanks for the update, though!
